CONTRIBUTORS.rst

```diff
@@ -115,6 +115,7 @@ Listed in alphabetical order.
   Emanuel Calso            `@bloodpet`_                  @bloodpet
   Eraldo Energy            `@eraldo`_
   Eric Groom               `@ericgroom`_
+  Ernesto Cedeno           `@codnee`
```

Comment:
If you want this to be a link to your profile (like the others) you should 

1) add an underscore after:

   ```suggestion
     Ernesto Cedeno           `@codnee`_
   ```

2) and define [the target later in the file](https://github.com/codnee/cookiecutter-django/blob/python_3.8/CONTRIBUTORS.rst#L273) (by adding this`.. _@codnee: https://github.com/codnee`)